resources/language/resource.language.zh_cn/strings.po

```diff
@@ -1053,40 +1053,40 @@ msgstr "默认首选立体声音轨"
 # Unused 30502 to 30599
 msgctxt "#30600"
 msgid "ESN / Widevine settings"
-msgstr ""
+msgstr "ESN / Widevine 设置"
 
 msgctxt "#30601"
 msgid "ESN:"
-msgstr ""
+msgstr "ESN："
 
 msgctxt "#30602"
 msgid "Change ESN"
-msgstr ""
+msgstr "更改ESN"
 
 msgctxt "#30603"
 msgid "Save system info"
-msgstr ""
+msgstr "保存系统信息"
 
 msgctxt "#30604"
 msgid "Widevine - Force security level"
-msgstr ""
+msgstr "Widevine - 强制安全级别"
 
 msgctxt "#30605"
 msgid "Force {}"
-msgstr ""
+msgstr "Force {}"
```

Comment:
do you need help for this?

Reply:
no answer...
@Liqianyu can you say something about this? i keep it as is?

Reply:
there will be a new release between 1 or 2 days
if I get no answer I will leave this value empty, it can be translated in the future